skydoc/main.py

```diff
@@ -133,14 +133,15 @@ class MarkdownWriter(object):
 
   def __init__(self, writer_options):
     self.__options = writer_options
+    self.__env = _create_jinja_environment(self.__options.link_ext)
 
   def write(self, rulesets):
     """Write the documentation for the rules contained in rulesets."""
     try:
       temp_dir = tempfile.mkdtemp()
       output_files = []
       for ruleset in rulesets:
-        if len(ruleset.rules) > 0:
+        if ruleset.rules or ruleset.macros:
```

Comment:
We should also handle repository rules here. Can you add a method `empty` to ruleset that returns True if there aren't any rules, macros, or repository rules?